blog/2025-01-28-ratify-1.4.0.mdx

```diff
@@ -0,0 +1,56 @@
+---
+slug: ratify-v1.4.0-is-available
+title: Announcing Ratify v1.4.0 - Revocation Checking with CRL Support, Enhanced Out-of-box Experience, and New Cloud Provider Support
+authors: [ratify]
+tags: [verification]
+---
+
+We are thrilled to announce the release of [Ratify v1.4.0](https://github.com/ratify-project/ratify/releases/tag/v1.4.0)! This milestone release introduces significant new features that enhance Ratify's capabilities as a trusted supply chain security tool. As always, we deeply appreciate the contributions from the community, which make these advancements possible.
+
+## New Features
+
+### Revocation Checking with Certificate Revocation List (CRL) Support
+   
+Security in supply chain verification is critical, and this release marks an important step forward with the addition of revocation checking. Certificate revocation checking enhances security by ensuring that compromised or expired certificates are not used, thereby maintaining the integrity and trustworthiness of digital signatures. Ratify supports validating signing identity (certificate and certificate chain) revocation status. 
+
+The CRL implementation uses Notation libraries and follows the [Notary Project specification](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#crls). Since [Notation v1.3.0](https://notaryproject.dev/blog/2025/announcing-notation-v1-3/), Ratify users can now use revocation checking by default if your images are signed by Notation since [CRL files are cached by default](https://notaryproject.dev/docs/user-guides/how-to/directory-structure/#caches). It enables end-to-end revocation checking for validating image signature since reduced network traffic to the CRL server which minimizes potential server throttling and enhanced performance with lower latency. See [Certificate Revocation Check (CRL) guidance](https://ratify.dev/docs/plugins/verifier/notation#certificate-revocation-check-crl) for details.

```

Comment:
I think "since CRL files are cached by default " can be removed, CRL is supported regardless of cache configuration.  We can add a seperate note that cache is enabled by default to enhance performance.